src/phoenix/server/api/routers/v1/spans.py

```diff
@@ -1119,3 +1121,129 @@ def convert_api_span_for_insertion(api_span: Span) -> SpanForInsertion:
         total_received=total_received,
         total_queued=len(spans_to_queue),
     )
+
+
+@router.delete(
+    "/projects/{project_identifier}/spans/{span_identifier}",
```

Comment:
remove the need for project to make this more convenient
```suggestion
    "/spans/{span_identifier}",
```